src/cmd-cloud-prune

```diff
@@ -114,14 +116,21 @@ def main():
             continue
         duration = convert_duration_to_days(policy[stream][action])
         ref_date = today_date - relativedelta(days=int(duration))
+        pruned_build_ids = []
 
         print(f"Pruning resources of type {action} older than {policy[stream][action]} ({ref_date.date()}) on stream {stream}")
         # Enumerating in reverse to go from the oldest build to the newest one
         for build in reversed(builds):
             build_id = build["id"]
-            if action in build.get("policy-cleanup", []):
-                print(f"Build {build_id} has already had {action} pruning completed")
-                continue
+            if action in build.get("policy-cleanup", {}):
+                if action == "images":
+                    # If images in images-keep in the policy and the images_kept in build doesn't match
+                    # let's run the pruning again on new images and update the images-kept.
+                    if policy.get(stream, {}).get("images-keep", []) != build["policy-cleanup"]["images-kept"]:
+                        pass
```

Comment:
What are you trying to achieve with this `pass`?

Reply:
maybe you mean for the else block below to be indented?

Reply:
I see how this could be complicated. Updated.